recipes/pyflann/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# python source is bundled with flann source
+cd src/python
+cmake . -DLIBRARY_OUTPUT_PATH=$PREFIX/lib -DFLANN_VERSION="$PKG_VERSION"
+$PYTHON setup.py install
```

Comment:
Alternatively, a `run_test.sh` or `run_test.py` could be added for this purpose.